servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java

```diff
@@ -111,6 +111,9 @@ public LRFUEvictor(BlockMetadataManagerView view, Allocator allocator) {
    */
   private StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocation location,
       EvictionPlan plan, List<Map.Entry<Long, Double>> sortedCRF) {
+    Preconditions.checkNotNull(location);
+    Preconditions.checkNotNull(plan);
+    Preconditions.checkNotNull(sortedCRF);
```

Comment:
Usually validation for user input is enough when CheckForNull is also used (http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/CheckForNull.html). With CFN compiler can provide such guarantee.

I can remove these checks if you feel it's too strong.